kolibri/plugins/learn/assets/test/views/learn-index.spec.js

```diff
@@ -1,22 +1,40 @@
 /* eslint-env mocha */
 const Vue = require('vue-test');
-const _ = require('lodash');
+const VueRouter = require('vue-router');
 const assert = require('assert');
 const LearnIndex = require('../../src/views/index.vue');
 const makeStore = require('../util/makeStore');
+const coreBase = require('../util/core-base.vue');
+
+const router = new VueRouter({
```

Comment:
As part of my ongoing project to make the code more testable, it would be easier to make a reusable router test double if the `routes` in `app.js` were accessible to an external module.